rmgpy/molecule/atomtype.py

```diff
@@ -242,7 +242,8 @@ def getFeatures(self):
 
```

Comment:
This is not because of the current PR, but there's an unnecessary blank line here. Could you remove it?